aio_pika/abc.py

```diff
@@ -5,10 +5,16 @@
 from functools import singledispatch
 from types import TracebackType
 from typing import (
-    Any, AsyncContextManager, AsyncIterable, Awaitable, Callable, Dict,
+    TYPE_CHECKING, Any, AsyncContextManager, AsyncIterable, Awaitable, Callable, Dict,
     FrozenSet, Generator, Iterator, MutableMapping, NamedTuple, Optional, Set,
     Tuple, Type, TypeVar, Union,
 )
+try:
+    from typing import TypedDict
+except ImportError:
+    if not TYPE_CHECKING:
```

Comment:
I think you can not check it, just do the import

Reply:
pyright is complaining without it, cf: https://github.com/microsoft/pyright/issues/2193